MBBSEmu/HostProcess/Enums/UserStatus.cs

```diff
@@ -0,0 +1,37 @@
+namespace MBBSEmu.HostProcess.Enums
+{
+    /// <summary>
+    /// User Status values, mostly taken from BRKTHU.H
+    /// </summary>
+    public enum UserStatus
+    {
+        UNUSED = 0, // default unused value
```

Comment:
You can use XML documentation on the enums to provide tool tips for their value while in an IDE